modules/affile/affile-source.c

```diff
@@ -330,95 +331,149 @@ affile_sd_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_options,
 }
 
 static gboolean
-affile_sd_init(LogPipe *s)
+_configure_loaded_logreader_from_persist_file(LogPipe *s, GlobalConfig *cfg)
 {
   AFFileSourceDriver *self = (AFFileSourceDriver *) s;
```

Comment:
I have two notes regarding this function:

1) the LogReader is never persisted to the disk, only kept as an object accross reloads (see the difference between PersistState vs. PersistConfig). I agree that the name of PersistConfig is not the best (it used to be the same object in the past), but still I wouldn't spread false information that the LogReader would be read from the persist file, which it isn't

2) also, I see the opportunity to merge logreader initialization after reload & when a new logreader is created, and name this function like _configure_reader() or _setup_reader() or merely _init_reader(). For this to be workable, the LogReader instance should be created in the affile_dd_init() function in case persist-config doesn't have one, and remove the similar code from _reopen_file()